vaadin-maps-leaflet-flow/src/main/java/software/xdev/vaadin/maps/leaflet/flow/data/LLayerGroup.java

```diff
@@ -42,7 +42,15 @@ protected String getLeafletComponentName()
 	public String buildClientJSItems() throws JsonProcessingException
 	{
 		// check LMap.addLComponent to understand why we do let item
-		StringBuilder jsCode = new StringBuilder("let item = L." + getLeafletComponentName() + "()");
+		StringBuilder jsCode;
+		if (getLeafletComponentName() == "markerClusterGroup"){
+			jsCode = new StringBuilder("let item = L." + getLeafletComponentName() + "({\n"
+				+ "maxClusterRadius: " + this.clusterRadius + "\n"
```

Comment:
Because it does not make sense that LLayoutGroup has a cluster option. It would be cleaner if we had and option object or string that you put here and you would override it in MCG.